### ISSUE TITLE
[Giphy] ☂️ Image recreation + prototype

### ISSUE THREAD

**Issue by thyhmdo** [carbon-design-system/carbon-website#4048]
### Summary
Ref issue https://github.com/carbon-design-system/carbon-website/pull/4000
After adding the missing play/pause function on the website's gifs, there are a few gif images that need to be recreated. Below is the task list. This work will close this [umbrella issue](https://github.com/carbon-design-system/carbon-website/issues/2683). 

```[tasklist]
### Tasks
- [ ] https://github.com/carbon-design-system/carbon-website/issues/4019
- [ ] https://github.com/carbon-design-system/carbon-website/issues/4020
- [ ] https://github.com/carbon-design-system/carbon-website/issues/4018
- [ ] https://github.com/carbon-design-system/carbon-website/issues/4051
- [ ] https://github.com/carbon-design-system/carbon-website/issues/4052
```
